docs/crud/subscriptions.md

```diff
@@ -53,6 +49,18 @@ subscription {
 }
 ```
 
+and with a filter:
```

Comment:
Worth adding that this is an equality filter? Pending https://github.com/aerogear/graphback/issues/1760

Reply:
I'll add it before Examples.